internal/pkg/git/git.go

```diff
@@ -90,9 +90,9 @@ func Checkout(ctx context.Context, url, ref string, sparse bool, logger log.Logg
 	}
 
 	// Clone parameters
-	params := []string{"clone", "-q", "--branch", r.ref}
+	params := []string{"clone", "-q", "--depth=1", "--branch", r.ref}
```

Comment:
A toto tiez pomohlo ... `depth=1` (clone bez git historie) sa moze pouzit vzdy, nie iba pre `sparse clone`.

Reply:
To jsem myslím řešil že jsem tam měl na začátku, ale něco tam potom nefungovalo. 🤔 Není to třeba pro checkout tagu? Branche asi ne, to nám funguje, ale možná ten tag. Podívám se na to. (A nebo kdyby to fakt nefungovalo tak můžeme říct, že podporujeme jen branche a ne tagy 🙂)

Reply:
Počkat asi ne, teď se dívám níž že už jsme to tam měli. Tak nic, dobrý, ignoruj mě. 😄